brave/src/main/java/brave/internal/handler/NoopAwareSpanHandler.java

```diff
@@ -75,10 +85,18 @@ public static SpanHandler create(SpanHandler[] handlers,
   }
 
   static final class CompositeSpanHandler extends SpanHandler {
+    final long handlesAbandonedBitset;
     final SpanHandler[] handlers;
 
     CompositeSpanHandler(SpanHandler[] handlers) {
       this.handlers = handlers;
+      long handlesAbandonedBitset = 0;
```

Comment:
Is the bitset to allow `handlesAbandoned` to be somewhat slow (maybe reading from a config value). Pushing that to the user may not be a big deal since I think it's pretty common for frameworks to read these sort of methods inline, even in JDK.

Reply:
sounds fine to me